server/worker_manager.js

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2017 The Absolute Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import cluster from 'cluster';
+import os from 'os';
+
+cluster.schedulingPolicy = cluster.SCHED_RR;
```

Comment:
What kind of schedulingPolicy is there?
Please add a comment with reference link here.